section2/exercises/methods.rb

```diff
@@ -42,4 +41,4 @@ def print_name(name)
 # YOUR ANSWER HERE
 
 # What did you name each parameter, and why?
-# YOUR ANSWER HERE
\ No newline at end of file
+# YOUR ANSWER HERE
```

Comment:
After working on the method you missed above, think about this question as well.